transform/mattermost-analytics/models/intermediate/product/configs/int__config_plugin.sql

```diff
@@ -0,0 +1,323 @@
+    , null as enable_mattermost_calls
+    , null as enable_mattermost_hackerone
+    , null as enable_mattermost_msteams_meetings
+    , null as enable_mattermost_msteams_sync
+    , null as enable_mattermost_profanity_filter
+    , null as enable_mattermost_servicenow
+    , null as enable_mattermost_servicenow_virtual_agent
+    , null as enable_memes
+    , enable_mscalendar
+    , enable_nps
+    , enable_nps_survey
+    , null as enable_playbooks
+    , enable_remote_marketplace
+    , null as enable_ru_loop_plugin_embeds
+    , null as enable_ru_loop_plugin_scheduler
+    , null as enable_ru_loop_plugin_user_fields
+    , null as enable_set_default_theme
+    , enable_skype4business
+    , enable_todo
+    , enable_uploads
+    , enable_webex
+    , enable_welcome_bot
+    , enable_zoom
+    , is_default_marketplace_url
+    , require_plugin_signature
+    , signature_public_key_files
+    , null as version_alertmanager
+    , version_antivirus
+    , version_autolink
+    , version_aws_sns
+    , null as version_bitbucket
+    , null as version_channel_export
+    , null as version_circleci
+    , null as version_confluence
+    , version_custom_user_attributes
+    , null as version_diceroller
+    , null as version_digitalocean
+    , null as version_focalboard
+    , null as version_giphy
+    , version_github
+    , version_gitlab
+    , null as version_icebreaker
+    , null as version_incident_management
+    , null as version_incident_response
+    , version_jenkins
+    , null as version_jespino_recommend
+    , version_jira
+    , null as version_jitsi
+    , null as version_matterpoll
+    , null as version_mattermost_agenda
+    , null as version_mattermost_apps
+    , null as version_mattermost_azure_devops
+    , null as version_mattermost_calls
+    , null as version_mattermost_hackerone
+    , null as version_mattermost_msteams_meetings
+    , null as version_mattermost_msteams_sync
+    , null as version_mattermost_profanity_filter
+    , null as version_mattermost_servicenow
+    , null as version_mattermost_servicenow_virtual_agent
+    , null as version_memes
+    , null as version_mscalendar
+    , version_nps
+    , null as version_playbooks
+    , null as version_set_default_theme
+    , null as version_skype4business
+    , null as version_todo
+    , version_webex
+    , version_welcome_bot
+    , version_zoom
+    , context_library_name
+    , context_library_version
+    , sent_at
+    , original_timestamp
+    , has_segment_telemetry_data
+    , null as has_rudderstack_telemetry_data
+from {{ ref('int_mattermost2__config_plugin') }}
+union
+select event_id
+    , event_table
+    , event_name
+    , server_id
+    , received_at
+    , timestamp
+    , installation_id
+    , anonymous_id
+    , server_ip
+    , allow_insecure_download_url
+    , automatic_prepackaged_plugins
+    , chimera_oauth_proxy_url
+    , enable_plugin
+    , enable_alertmanager
+    , enable_antivirus
+    , coalesce(enable_autolink,
```

Comment:
These coalesce expressions can be moved to the staging model. Then `dbt_utils.union_relations` can be used in this model.